### PR TITLE
android target link fixed

### LINKED ISSUE #9595: Change link target for Android incompatibility warning
When Fenix users visit a listing page on AMO, they see this warning: 

Screenshot_20200826-114132

We link to the SUMO article targeted for Firefox Preview users. Since Preview has graduated, we should update the link to https://support.mozilla.org/kb/find-and-install-add-ons-firefox-android. 

This should be included in our next push. :)

### PR DESCRIPTION
**Pull request by navdhakar**
Fixes #9595 

android sumo link fixed

### CODE REVIEW

**Comment by championshuttler** [comment]
Oops there is already a PR #9598 for the issue by @muffinresearch

File: src/amo/components/WrongPlatformWarning/index.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ import type { ReactRouterLocationType } from 'core/types/router';
 import './styles.scss';
 
 export const FENIX_LINK_DESTINATION =
-  'https://support.mozilla.org/kb/add-compatibility-firefox-preview/';
+   'https://support.mozilla.org/kb/find-and-install-add-ons-firefox-android';
```
Comment (championshuttler) [inline src/amo/components/WrongPlatformWarning/index.js:31]: Nit: you added a blank space at starting of the line, please remove it

**Comment by muffinresearch** [comment]
If you can fix the leading space we can take this patch, I've closed my PR 👍

**Review by navdhakar — COMMENTED** [review]
extra space removed

**Review by championshuttler — COMMENTED** [review]
looks good to me thank you so much @navdhakar 🎉

**Comment by championshuttler** [comment]
@navdhakar oops did you close the PR by mistake?

**Comment by navdhakar** [comment]
thank you so much..

**Review by muffinresearch — APPROVED** [review]
Looks good, thanks for the patch.

**Comment by caitmuenster** [comment]
Thanks so much for the patch, @navdhakar! 🎉 Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition). 

Welcome onboard! I look forward to seeing you around the project.

**Comment by navdhakar** [comment]
Thanks i'll be happy to contribute

**Pull request merged**